src/gamekit.cs

```diff
@@ -2964,10 +2992,20 @@ interface GKLeaderboardEntry {
 		[Export ("date", ArgumentSemantic.Strong)]
 		NSDate Date { get; }
 #endif
-		[NoWatch] // header lists watch as supported, but UIViewController is not available on Watch!
+
+#if XAMCORE_5_0
+		[TV(17, 0), iOS(17, 0), MacCatalyst(17, 0), Mac(14,0)]
+		[Export("challengeComposeControllerWithMessage:players:completion:")]
```

Comment:
This needs to be bound as a new method, not replace an existing one in the future, because they have different signatures (the callbacks are different).